src/IssueViz/TableControls/IssueTablesSelectionMonitor.cs

```diff
@@ -42,21 +41,18 @@ internal interface IIssueTablesSelectionMonitor
     internal class IssueTablesSelectionMonitor : IIssueTablesSelectionMonitor
     {
         private readonly IAnalysisIssueSelectionService selectionService;
-        private readonly IAnalysisIssueVisualizationConverter converter;
 
         [ImportingConstructor]
-        public IssueTablesSelectionMonitor(IAnalysisIssueSelectionService selectionService, IAnalysisIssueVisualizationConverter converter)
+        public IssueTablesSelectionMonitor(IAnalysisIssueSelectionService selectionService)
```

Comment:
This class handles selection change notifications from the error list. Previously it was being handed an issue which it converted to a issue viz. Now it's being handed an issue viz, so it doesn't need to do the conversion.